amp.php

```diff
@@ -667,6 +668,7 @@ function amp_render() {
  * @global WP_Query $wp_query
  */
 function amp_render_post( $post ) {
+	_deprecated_function( __FUNCTION__, '1.5' );
```

Comment:
We need to check if these functions are being used in themes/plugins, and if so, potentially defer this hard deprecation until we have the AMP middleware rendering stack available to use instead (same for `AMP_Content`).

Reply:
No relevant usages: https://wpdirectory.net/search/01DT152D0C97YRHGGJHYQ19X5X